python/lsst/afw/cameraGeom/fitsUtils.py

```diff
@@ -180,7 +182,7 @@ def _makeDefaultAmpMap(self):
                    ('GAIN', 'setGain', 1.),
                    ('RDNOISE', 'setReadNoise', 0.),
                    ('SATURATE', 'setSaturation', 2<<15),
-                   ('RDCRNR', 'setReadoutCorner', afwTable.LL),
+                   ('RDCRNR', 'setReadoutCorner', int(afwTable.ReadoutCorner.LL), afwTable.ReadoutCorner),
```

Comment:
@pschella did improve pybind11 support for enums but I don't remember the details. This change was definitely required to avoid type errors.